data/human/free worlds 0 prologue.txt

```diff
@@ -147,15 +147,16 @@ mission "Pact Recon 3"
 		attributes "dirt belt" "near earth" "rim" "south"
 		not government "Pirate"
 	destination "Glaze"
-	waypoint "Boral"
-	waypoint "Hintar"
-	waypoint "Naper"
-	waypoint "Orvala"
+	mark "Boral"
+	mark "Hintar"
+	mark "Naper"
+	mark "Orvala"
 	to offer
 		random < 40
 		has "Pact Recon 2: done"
 		not "event: war begins"
-	
+	to complete
```

Comment:
`to complete` above `on complete`, `on complete` as the last action in the mission (i.e. move the `on enter`s above the `on visit`).